terraform/executeTerraformTest.sh

```diff
@@ -54,6 +54,9 @@ case "$service" in
     ;;
     "EKS_ADOT_OPERATOR") TEST_FOLDER="./eks/";
     ;;
+    "EKS_ADOT_OPERATOR_ARM64") TEST_FOLDER="./eks/"
+        opts+=" -var=eks_cluster_name=arm64-adot-op-cluster";
+    ;;
```

Comment:
```suggestion
    "EKS_ADOT_OPERATOR_ARM64") TEST_FOLDER="./eks/"
        export TF_VAR_eks_cluster_name=arm64-adot-op-cluster;
    ;;
```
I think we should stay consistent with how we set the `eks_cluster_name` variable in this file.

Reply:
The issue here is that for operator tests, `eks_cluster_name` is passed in through a `.tfvars` file (see [here](https://github.com/aws-observability/aws-otel-test-framework/blob/terraform/terraform/executeTerraformTest.sh#L31) or [this example](https://github.com/aws-observability/aws-otel-test-framework/blob/terraform/terraform/testcases/prometheus_static_adot_operator/parameters.tfvars#L10)). This takes [precedence](https://www.terraform.io/language/values/variables#variable-definition-precedence) over environment variables, so we cannot do an `export TF_VAR_...` for setting the arm64 operator cluster name

Reply:
To add on to this, the `-var-file` option that provides `parameters.tfvars` would be provided before the `-var=eks_cluster_name` option, which would follow the correct order of precedence for us to set `arm64-adot-op-cluster` as the name

Reply:
I feel a little conflicted on this choice. On one hand I understand why the env var won't work, but on the other it's a little confusing to look at from an outsiders perspective. I would be wondering "why is it done one way here and a different way below". 

I think we have two options
1. Add some documentation explaining this choice, a comment will probably do
2. Remove the cluster name from the existing parameters files so that we can uniformly use env vars. 

I think another downside to this is that we have a hard dependency for our arm-64 cluster names to use this specific name. It's not very portable between accounts but I am not sure if we are going for portability with this script. To be fair, all EKS tests other than the `EKS_ARM64` tests have this issue because we are using default values from the terraform modules.

Reply:
Update here, I've filed #784 that changes the usage of environment variables to set terraform variables in the script to use the `-var` option instead. It also addresses a few other things as well. The hard dependency for operator/arm64 operator tests to use these cluster names still exists, but it's slightly better than having `eks_cluster_name` set in multiple `parameters.tfvars` files.